server/cmd/code-annotation/serializer/serializers.go

```diff
@@ -0,0 +1,112 @@
+package serializer
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
```

Comment:
render package is quite complex. The main advantage of it, you can handle different types of input/output. Like json/xml for example. But I'm not sure we really need it in this project.
Are you open to considering simpler solution?

Reply:
It was quite easy to implement in our case, but I'd reconsider it for sure, and ping you when done :dagger: Thanks for being nitpicker !!!